Sources/TensorFlow/Core/LazyTensorOperation.swift

```diff
@@ -682,7 +682,7 @@ extension LazyTensor: CustomStringConvertible {
 
 extension LazyTensorOperation: CustomStringConvertible {
     public var description: String {
-        let attrsDesc = attrs.map { (name, value) in "\(name): \(value)" }
+        let attrsDesc = attrs.keys.sorted(by: <).map { "\($0): \(attrs[$0]!)" }
```

Comment:
Please use [`Dictionary.sorted(by:)`](https://developer.apple.com/documentation/swift/dictionary/2298238-sorted).
```suggestion
        let attrsDesc = attrs.sorted(by: { $0.key < $1.key }).map { "\($0): \($1)" }
```